readthedocs/builds/views.py

```diff
@@ -82,6 +84,30 @@ class BuildDetail(BuildBase, DetailView):
     def get_context_data(self, **kwargs):
         context = super(BuildDetail, self).get_context_data(**kwargs)
         context['project'] = self.project
+        scheme = ("https://github.com/rtfd/readthedocs.org/issues/new"
+                  "?title={title}{build_id}"
+                  "&body={body}")
+
+        body = ("# Details:\n\n"
```

Comment:
Is better if you use the multistring syntax here (`"""`), so it's more clear and you will not need to add `\n`.

Reply:
Removing the ``\n`` creates trouble when ``quote`` is used to render url. Newlines and spaces are not converted properly. So, I used the ``\n``.

Reply:
If you're having trouble with spaces, consider https://docs.python.org/2/library/textwrap.html#textwrap.dedent

Reply:
Thanks for this @agjohnson, I have updated the string syntax!